lib/spi/types.ts

```diff
@@ -24,15 +26,251 @@ export type Values = {
     [key: string]: any;
 };
 
+/**
+ * Utility type for Karpenter requirement values
+ */
+export type reqValues = {
+    key: string,
```

Comment:
is there are reason why Karpenter specific type is plance in spi as opposed to a regular export from index.ts?

Reply:
moved the types only relevant to Karpenter back to `index.ts`